resources/views/page/overview.blade.php

```diff
@@ -4,14 +4,16 @@
 @section('description', $page->meta_description)
 
 @section('content')
-    <div class="container mx-auto">
-        @if($page->identifier != 'home' && $page->content_heading)
-            <h1 class="font-bold text-4xl mb-5">{{ $page->content_heading }}</h1>
+    <div class="container">
+        @if ($page->identifier != 'home' && $page->content_heading)
+            <h1 class="mb-5 text-4xl font-bold">{{ $page->content_heading }}</h1>
         @endif
-        @includeIf('pages.'.$page->identifier)
-        @widget('content', 'pages', ($page->identifier == 'home' ? 'cms' : $page->identifier).'_index_index')
-        @if($page->content)
-            <div class="mb-5 prose prose-green">
+        @includeIf('pages.' . $page->identifier)
+        <div class="hidden lg:block">
+            @widget('content', 'pages', ($page->identifier == 'home' ? 'cms' : $page->identifier) . '_index_index')
+        </div>
```

Comment:
So we don't want this on mobile as it's not responsive, but do we still want this on desktop?